scene/gui/tab_container.cpp

```diff
@@ -682,7 +681,7 @@ Vector<Control *> TabContainer::_get_tabs() const {
 	Vector<Control *> controls;
 	for (int i = 0; i < get_child_count(); i++) {
 		Control *control = Object::cast_to<Control>(get_child(i));
-		if (!control || control->is_top_level_control()) {
+		if (!control || control->is_set_as_top_level()) {
```

Comment:
This one looks like a typo made in https://github.com/godotengine/godot/commit/fc9077a213e8a73137855b41fda99819e7d1af85, everywhere else `is_set_as_top_level` is used.